test/controllers/organizations/adopter_fosterer/fostered_pets_controller_test.rb

```diff
@@ -0,0 +1,46 @@
+require "test_helper"
+require "action_policy/test_helper"
+
+class Organizations::AdopterFosterer::FosteredPetsControllerTest < ActionDispatch::IntegrationTest
+  context "authorization" do
+    include ActionPolicy::TestHelper
+
+    setup do
+      @organization = ActsAsTenant.current_tenant
+      @fosterer = create(:fosterer)
+      sign_in @fosterer
+    end
+
+    context "#index" do
+      should "be authorized" do
+        get adopter_fosterer_fostered_pets_url
+        assert_response :success
+      end
+
+      should "have authorized scope" do
+        ActsAsTenant.with_tenant(@organization) do
+          assert_have_authorized_scope(
+            type: :active_record_relation,
+            with: Organizations::AdopterFosterer::MatchPolicy
+          ) do
+            get adopter_fosterer_fostered_pets_url
+          end
+        end
+      end
+
+      should "return only foster matches for the person" do
```

Comment:
Per my comment above - lets update this to make sure only current fosters are returned for now.